articles/migrations/guides/calling-api-with-idtokens.md

```diff
@@ -0,0 +1,254 @@
+
+Note that the last two endpoints are used for [Account Linking](/link-accounts#the-management-api). The changes in this functionality are the following:
+- You can no longer use an ID Token at the `Authorization` header
+- If you already use Access Tokens at the `Authorization` header you don't need to change something in your implementation, but do note that now you can send at the request's body the secondary account's ID Token, instead of the `user_id` (or you can keep using the `user_id`)
+
+**Nothing else changes in how these endpoints work**. You should expect the same request and response schemas and only need update the token that you use for authorization.
+
+## Changes in scopes
+
+The actions you can perform with the Management API depend on the [scopes](/scopes#api-scopes) that your Access Token contains. With this migration you can either get a "limited" Access Token that can update only the logged-in user's data, or an Access Token that can update the data of any user. In the following matrix you can see the scopes that your token needs to have per case and per endpoint.
+
+| **Endpoint** | **Scope for current user** | **Scope for any user** |
+|-|-|
+| [GET /api/v2/users/{id}](/api/management/v2#!/Users/get_users_by_id) | `read:current_user` | `read:users` |
+| [GET /api/v2/users/{id}/enrollments](/api/management/v2#!/Users/get_enrollments) | `read:current_user` | `read:users` |
+| [POST/api/v2/users/{id}/identities](/api/management/v2#!/Users/post_identities) | `update:current_user_identities` | `update:users` |
+| [DELETE /api/v2/users/{id}/identities/{provider}/{user_id}](/api/management/v2#!/Users/delete_provider_by_user_id) | `update:current_user_identities` | `update:users` |
+| [PATCH /api/v2/users/{id}](/api/management/v2#!/Users/patch_users_by_id) | `update:current_user_metadata` | `update:users` |
+| [PATCH /api/v2/users/{id}](/api/management/v2#!/Users/patch_users_by_id) | `create:current_user_metadata` | `update:users` |
+| [DELETE /api/v2/users/{id}/multifactor/{provider}](/api/management/v2#!/Users/delete_multifactor_by_provider) | `delete:current_user_metadata` | `update:users` |
+| [POST /api/v2/device-credentials](/api/management/v2#!/Device_Credentials/post_device_credentials) | `create:current_user_device_credentials` | `create:device_credentials` |
+| [DELETE /api/v2/device-credentials/{id}](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id) | `delete:current_user_device_credentials` | `delete:device_credentials` |
+
+For example, if I get an Access Token that contains the scope `read:users` I can retrieve the data of **any user** using the [GET /api/v2/users/{id} endpoint](/api/management/v2#!/Users/get_users_by_id). But if my token contains the scope `read:current_user` I can only retrieve the information of the **currently logged-in user** (the one that the token was issued for).
```

Comment:
This is the story, not a standalone example. Current customers are using id_tokens as auth to modify the details of a particular user. This entire migration is to move these customers to use access_tokens as auth with the `*:current_user` scopes.